tasks/update-granules-cmr-metadata-file-links/index.js

```diff
@@ -102,7 +112,8 @@ async function updateGranulesCmrMetadataFileLinks(event) {
     cmrGranuleUrlType,
     config.distribution_endpoint,
     bucketTypes,
-    distributionBucketMap
+    distributionBucketMap,
+    excludeFileRegexPattern
```

Comment:
Actually `excludeFileRegex` is not a nested config field, so we can just pass `config.excludeFileRegexPattern` to this function like `config.distribution_endpoint`